### PR TITLE
Split webUISharingInternalUsers into two suites

### PR DESCRIPTION
**Pull request by dpakach**
## Description

Split acceptance test suite `webUISharingInternalUsers` into two suites.

## Related Issue

- Fixes https://github.com/owncloud/QA/issues/621

## Motivation and Context

This suite contains features regarding `share autocompletion` and `share with users`. These are unrelated features and contain large number of scenarios and required a lot of time to complete in CI. So they needed to be split into their respective suites.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

**Review by skshetry — COMMENTED** [review]
If tests pass, :+1:

File: .drone.yml [hunk 1]
```diff
@@ -1281,6 +1281,19 @@ matrix:
...
+      BROWSER: chrome
+      BEHAT_SUITE: webUISharingAutocompletion
+      DB_TYPE: mariadb
+      USE_SERVER: true
+      SERVER_PROTOCOL: https
+      INSTALL_SERVER: true
+      CHOWN_SERVER: true
+      OWNCLOUD_LOG: true
+      INSTALL_TESTING_APP: true
+      USE_EMAIL: true
```
Comment (paurakhsharma) [inline .drone.yml]: Email is not required here.

File: tests/acceptance/config/behat.yml [hunk 2]
```diff
@@ -386,6 +386,18 @@ default:
...
+    webUISharingAutocompletion:
+      paths:
+        - '%paths.base%/../features/webUISharingAutocompletion'
+      contexts:
+        - FeatureContext: *common_feature_context_params
+        - WebUIFilesContext:
+        - WebUIGeneralContext:
+        - WebUILoginContext:
+        - WebUISharingContext:
+        - EmailContext:
```
Comment (paurakhsharma) [inline tests/acceptance/config/behat.yml]: Email is not required here.

**Review by paurakhsharma — COMMENTED** [review]
Looks good to me :+1:

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
`user_ldap` will need to know about the new suite tomorrow

**Comment by phil-davis** [comment]
Backport `stable10` #35411

**Comment by phil-davis** [comment]
Rebased to get good CI

**Pull request merged**